public/components/application-ui-tables/8.html

```diff
@@ -0,0 +1,47 @@
+<div class="overflow-x-auto rounded-lg border border-gray-200">
+  <table class="min-w-full divide-y-2 divide-gray-200 bg-white text-sm">
+    <thead class="ltr:text-left rtl:text-right">
+      <tr>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Name 
+            <span class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Date of Birth 
+            <span class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Role 
+            <span class="sort-icon">&uarr;&darr;</span>
+        </th>
+        <th class="p-2 text-left cursor-pointer whitespace-nowrap px-4 py-2 font-medium text-gray-900">
+            Salary 
+            <span class="sort-icon">&uarr;&darr;</span>
```

Comment:
I think we should use a chevron here:

```
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-6">
  <path stroke-linecap="round" stroke-linejoin="round" d="m19.5 8.25-7.5 7.5-7.5-7.5" />
</svg>
```

Might need some tweaking as `size-6` is pretty large.

Reply:
No need for the `sort-icon` class either.